ros2topic/ros2topic/verb/hz.py

```diff
@@ -86,6 +87,7 @@ def main(self, *, args):
         return main(args)
 
 
+<<<<<<< HEAD
```

Comment:
In case you didn't notice, there's still some merge conflicts here

Reply:
Missed that, thanks.